src/utils/api/btc/transactions.js

```diff
@@ -1,89 +1,83 @@
-import * as bitcoin from 'bitcoinjs-lib';
-import * as popsicle from 'popsicle';
+import bitcoin from 'bitcoinjs-lib';
```

Comment:
This still works like this?? Because I remember that this version of `bitcoinjs-lib` don't export a default, so it was needed to import as `* as bitcoin`

Reply:
Interesting, I had a problem with it at first, but now it seems fine. It's changed because I got the latest version from Altay that is compatible with the latest bitcoin-bridge.